tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+# Evaluation of Microsoft Playwright
+
+## Introduction
+**Automation testing**
```

Comment:
Is `**Automation testing**` a subsection here? It's not a big deal, but the text says "Automation testing Automation testing..." 😄

Reply:
Got it. Thanks